app/src/layouts/settingsLayout/footer/footer.jsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2023 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Fragment } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { connect } from 'react-redux';
+import { referenceDictionary } from 'common/utils';
+import { uiBuildVersionSelector } from 'controllers/appInfo';
+import { instanceTypeSelector } from 'controllers/appInfo/selectors';
+import { EPAM, SAAS } from 'controllers/appInfo/constants';
+import styles from './footer.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  github: {
+    id: 'SettingsLayout.footer.github',
+    defaultMessage: 'Fork us on GitHub',
+  },
+  slack: {
+    id: 'SettingsLayout.footer.slack',
+    defaultMessage: 'Chat with us on Slack',
+  },
+  contactUs: {
+    id: 'SettingsLayout.footer.contactUs',
+    defaultMessage: 'Contact us',
+  },
+  epam: {
+    id: 'SettingsLayout.footer.epam',
+    defaultMessage: 'EPAM',
+  },
+  documentation: {
+    id: 'SettingsLayout.footer.documentation',
+    defaultMessage: 'Documentation',
+  },
+  privacyPolicy: {
+    id: 'SettingsLayout.footer.privacyPolicy',
+    defaultMessage: 'Privacy Policy',
+  },
+  rights: {
+    id: 'SettingsLayout.footer.rights',
+    defaultMessage: '© Report Portal {currentYear} All rights reserved',
+  },
+  build: {
+    id: 'SettingsLayout.footer.build',
+    defaultMessage: 'Build: {buildVersion}',
+  },
+});
+
+const FooterBase = ({ buildVersion, instanceType, date }) => {
+  const { formatMessage } = useIntl();
+
+  const currentYear = date.getFullYear();
```

Comment:
It looks like the date is not actually a prop and is never passed through props.
Why not calculate the actual year right here?

```suggestion
  const currentYear = (new Date()).getFullYear();
```